README.md

```diff
@@ -1,6 +1,18 @@
 # evidence
 
-Word2Vec-based assisted close reading tool with support for context-based search and concept-based search.
+Doc2Vec-based assisted close reading tool with support for abtract concept-based search and context-based search.
+
+## Machine supported research in humanities
+While research in the humanities has been able to leverage the digitization of text corpora and the development of computer based text analysis tools to its benefit, the interface current systems provide the user with is incompatible with the the proven method of scholarly close reading of texts which is key in many research scenarios pursuing complex research questions.
+
+What this boils down to, is the fact that it is often restrictive and difficult to formulate selection criteria, in particular for more complex or abstract concept, in the framework of a keyword based search which is the standard entry point to digitized text collections.
+
+## Querying by example - close reading with tailored suggestions  
+Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework evidence learns abstract representations of the content of the elements of the user's corpus.
```

Comment:
is it word2vec or doc2vec? either way we should pick one and use it consistently throughout.

Reply:
it is doc2vec in all cases. I'll give it a comb through

Reply:
```
$ grep -inr word2vec
Binary file experiments/demo/model/doc2vec.d2v matches
README.md:3:Word2Vec-based assisted close reading tool with support for context-based search and concept-based search.
README.md:50:## Generating the word2vec model
```

Reply:
thanks